src/extension/openFileAtLocation.ts

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for details.
+
+import {ExtensionMessage, MessagingChannel} from "../common/extensionMessaging";
+import {InterProcessMessageSender} from "../common/interProcessMessageSender";
+import {ReactNativeProjectHelper} from "../common/reactNativeProjectHelper";
+import {InternalErrorCode} from "../common/error/internalErrorCode";
+import {ErrorHelper} from "../common/error/errorHelper";
+import * as path from "path";
+
+/* Usage:
+...path\openFileAtLocation.js filename:lineNumber
+...path\openFileAtLocation.js filename
+*/
+
+try {
+    const fileInfo = process.argv[2].split(":");
+    const filename: string  = fileInfo[0];
+    let lineNumber: number = null;
+    if (fileInfo.length >= 2) lineNumber = parseInt(fileInfo[1]);
+
+    getReactNativeWorkspaceForFile(filename).then(projectRootPath => {
+        const remoteExtensionServerPath = new MessagingChannel(projectRootPath).getPath();
+        const interProcessMessageSender = new InterProcessMessageSender(remoteExtensionServerPath);
+        interProcessMessageSender.sendMessage(ExtensionMessage.OPEN_FILE_AT_LOCATION, [filename, lineNumber]);
```

Comment:
You should use RemoteExtension here instead (you'll have to add your new message)